parameters:
  p1_value: 0.2  # p1の値
  p2_value: 0.2  # p2の値
  angle: 0.0    # 角度